test/python/converters/test_circuit_to_dagdependency.py

```diff
@@ -14,11 +14,11 @@
 dependency."""
 
 import unittest
+from test import QiskitTestCase
 
 from qiskit.converters.dagdependency_to_circuit import dagdependency_to_circuit
 from qiskit.converters.circuit_to_dagdependency import circuit_to_dagdependency
 from qiskit import QuantumRegister, ClassicalRegister, QuantumCircuit
-from test import QiskitTestCase  # pylint: disable=wrong-import-order
```

Comment:
I can't say that I fully understand why PR #10998 replaced all lines of the form 
```
from test import QiskitTestCase
```
to
```
from test import QiskitTestCase  # pylint: disable=wrong-import-order
```
but it may be best to keep the change here.

Reply:
Not sure why either, but can leave as is.